test/test_utils.py

```diff
@@ -534,52 +585,68 @@ def test_single_keep(self):
-                                             batch_size=self.batch_size,
-                                             num_workers=0,
-                                             drop_last=True)
+        dataloader: DataLoader = DataLoader(
+            self.dataset,  # type: ignore[arg-type]
+            batch_size=self.batch_size,
+            num_workers=0,
+            drop_last=True,
+        )
         dataiter = iter(dataloader)
         self.assertEqual(len(list(dataiter)), 1)
 
-    @unittest.skip("FIXME: Intermittent CUDA out-of-memory error on Windows and time-out under ASAN")
+    @unittest.skip(
+        "FIXME: Intermittent CUDA out-of-memory error on Windows and time-out under ASAN"
+    )
     def test_multi_keep(self):
-        dataloader : DataLoader = DataLoader(self.dataset,  # type: ignore[arg-type]
-                                             batch_size=self.batch_size,
-                                             num_workers=2,
-                                             drop_last=False,
-                                             timeout=self.MAX_TIMEOUT_IN_SECOND)
+        dataloader: DataLoader = DataLoader(
+            self.dataset,  # type: ignore[arg-type]
+            batch_size=self.batch_size,
+            num_workers=2,
+            drop_last=False,
+            timeout=self.MAX_TIMEOUT_IN_SECOND,
+        )
         dataiter = iter(dataloader)
         self.assertEqual(len(list(dataiter)), 2)
 
     def test_multi_drop(self):
-        dataloader : DataLoader = DataLoader(self.dataset,  # type: ignore[arg-type]
-                                             batch_size=self.batch_size,
-                                             num_workers=2,
-                                             drop_last=True,
-                                             timeout=self.MAX_TIMEOUT_IN_SECOND)
+        dataloader: DataLoader = DataLoader(
+            self.dataset,  # type: ignore[arg-type]
+            batch_size=self.batch_size,
+            num_workers=2,
+            drop_last=True,
+            timeout=self.MAX_TIMEOUT_IN_SECOND,
+        )
         dataiter = iter(dataloader)
         self.assertEqual(len(list(dataiter)), 1)
 
 
 test_dir = os.path.abspath(os.path.dirname(str(__file__)))
 
 
-@unittest.skipIf('SKIP_TEST_BOTTLENECK' in os.environ.keys(), 'SKIP_TEST_BOTTLENECK is set')
+@unittest.skipIf(
+    "SKIP_TEST_BOTTLENECK" in os.environ.keys(), "SKIP_TEST_BOTTLENECK is set"
+)
 class TestBottleneck(TestCase):
     def _run(self, command, timeout=30):
         """Returns (return-code, stdout, stderr)"""
         import subprocess
 
-        p = subprocess.Popen(command, stdout=subprocess.PIPE,  # noqa: P204
-                             stderr=subprocess.PIPE, shell=True)
+        p = subprocess.Popen(
+            command,
+            stdout=subprocess.PIPE,  # noqa: P204
+            stderr=subprocess.PIPE,
+            shell=True,
+        )
```

Comment:
the location of `noqa` may be wrong

Reply:
Maybe it's unnecessary, let me try to remove it.